kernel_tracing/bpftrace_scripts/suspend_resume_time.bt

```diff
@@ -19,6 +19,7 @@ kretprobe:sof_resume, kretprobe:sof_suspend {
   // When the function returns, we use the saved timestamp to determine execution time
   printf("%s (%s) finished, took %d nsecs\n", probe, @type, nsecs - @start);
   @usecs[probe, @type] = hist((nsecs - @start) / 1000);
+  @avg[probe, @type] = avg((nsecs - @start) / 1000);
```

Comment:
I added it as part of that commit since the Python script can't operate on the histogram value, so it needs another value to use for the threshold checking. I think this will end up all squashed together anyways, but I can move it to the previous commit if you'd prefer.

Reply:
I think it'd be clearer if all bpf scripts were moved to the previous patch